src/components/footer/footer.module.scss

```diff
@@ -32,3 +36,56 @@
     @include typography.body1;
   }
 }
+
+.bottomBar {
+  display: flex;
+  padding-bottom: 32px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 32px;
+  @include fonts.poppins;
+  width: 1120px;
+
+  &_divider {
+    width: 100%;
+    height: 1px;
+    color: colors.$neutrals6;
+  }
+  &_content {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: flex-start;
+  }
+
+  &_leftText {
+    color: colors.$neutrals4;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 20px;
+  }
+  &_rightFrame {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 16px;
+  }
+  &_rightText {
+    color: colors.$neutrals2;
+    text-align: right;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 20px;
+  }
+  &_accept {
+    color: colors.$primary1;
+    text-align: right;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 20px;
```

Comment:
this should be probably defined as a mixin in typography, for example:

```ts
@mixin body1--bold {
  font-size: 24px;
  line-height: 32px;
  font-weight: 600;
}
```

see how this font is named in Figma, each one has its own name